HPXMLtoOpenStudio/resources/weather.rb

```diff
@@ -308,9 +308,11 @@ def calc_ground_temperatures
     bo = (data.MonthlyAvgDrybulbs.max - data.MonthlyAvgDrybulbs.min) * 0.5
 
     @data.GroundMonthlyTemps = []
+    adj_annual_avg_drybulb_c = 0.91 * UnitConversions.convert(data.AnnualAvgDrybulb, 'F', 'C') + 1.82
```

Comment:
If we end up sticking with this, we should add a comment explaining where this equation/regression came from.

Reply:
@shorowit : I added a comment in [d3eb3c4](https://github.com/NREL/OpenStudio-HPXML/pull/1475/commits/d3eb3c47860e8b45532d5ea5530f9e39570a44e8)